mkosi/resources/man/mkosi.1.md

```diff
@@ -256,6 +256,12 @@ Those settings cannot be configured in the configuration files.
     working directory. If the empty string is specified, all configuration in
     the current working directory will be ignored.
 
+`--default=`
+:   Takes an absolute path to a configuration file. **mkosi** parses this file
+    before doing anything. This allows a user to select an entry point to
+    programmatically decide among various available options (e.g.: profiles)
+    without having to select them on the command line.
+
```

Comment:
I do not quite understand how this is different from an `--include=`, especially if that include points to a directory with a `mkosi.local.conf` file

Reply:
--include is not used, so it's not useful

Reply:
No, backward compatibility must be maintained, that is staying exactly as it is